kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSessionImpl.scala

```diff
@@ -73,6 +77,39 @@ class KyuubiSessionImpl(
     case (key, value) => sessionConf.set(key, value)
   }
 
+  val (clusterManager, kubernetesInfo) = engine.engineType match {
```

Comment:
it is not accurate here

Reply:
For the `clusterManager`, you can leave it empty and update it after getting the engine info.